src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -291,10 +293,10 @@ private X509CertificateContainer CreateCertificate(bool isAuthority, bool isMach
                 var authorityKeyIdentifier = new AuthorityKeyIdentifier(
                     SubjectPublicKeyInfoFactory.CreateSubjectPublicKeyInfo(_authorityKeyPair.Public),
                     new GeneralNames(new GeneralName(authorityX509Name)),
-                    new BigInteger(7, _random).Abs());
+                    serialNum);
```

Comment:
Yes, this is to fix bug in certificate issuer serial number. Copy+paste issue that dates way back